src/app.ts

```diff
@@ -35,9 +35,15 @@ const DEFAULT_NOT_ALLOWED_METHOD = () => {
   throw new HttpError(405);
 };
 
-export type ListenOptions = Partial<Deno.ServeTlsOptions> & {
-  remoteAddress?: string;
-};
+export type ListenOptions =
+  & Partial<
+    Deno.ServeTcpOptions | (Deno.ServeTcpOptions & Deno.TlsCertifiedKeyPem)
```

Comment:
```suggestion
    Deno.ServeTcpOptions & Deno.TlsCertifiedKeyPem
```

In a `Partial`, this means the same thing.